retrieval/target.go

```diff
@@ -197,10 +197,24 @@ func (t *Target) Update(cfg *config.ScrapeConfig, baseLabels, metaLabels clientm
 
 	t.url.Scheme = cfg.Scheme
 	t.url.Path = string(baseLabels[clientmodel.MetricsPathLabel])
+	params := url.Values{}
+	for k, v := range cfg.Params {
+		params[k] = make([]string, len(v))
+		copy(params[k], v)
+	}
+	for k, v := range baseLabels {
+		if strings.HasPrefix(string(k), "__param_") {
```

Comment:
Vendoring - at least godep - is version agnostic. Only revisions matter. So this is fine and we've been doing this multiple times over the last few months to add constants.